### PR TITLE
Add contribute.json.

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@tofumatt r?

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        "clone": "https://github.com/mozilla/nocturnal.git"
+    },
+    "participate": {
+        "home": "https://github.com/mozilla/nocturnal",
+        "docs": "https://github.com/mozilla/nocturnal/blob/master/README.markdown",
+        "mailing-list": "https://www.mozilla.org/about/forums/#dev-mozilla-org",
+        "irc": "irc://irc.mozilla.org/#webdev",
+        "irc-contacts": [
+            "tofumatt",
+            "gavin"
```
Comment (tofumatt) [inline contribute.json]: Not sure gavin is an appropriate irc contact, he doesn't work on this much.

**Comment by tofumatt** [comment]
I'm not aware of what this is for, and it seems stuff that would be better in a human readable README, but if people actually use this I guess we can merge. 

What is the usual way people would consume this file?

File: contribute.json (hunk 1 above)
Comment (glogiotatidis) [inline contribute.json]: Sure I'll remove. I got this from the README

**Comment by glogiotatidis** [comment]
This is a structured way to add contribution information. I was just going through @peterbe's [list of projects without one](https://gist.github.com/peterbe/82bece77e4b26c1ecfc5) and I picked yours to add one :) 

For more see https://contribute.paas.allizom.org/

**Comment by tofumatt** [comment]
Fair enough, seems cool!

**Comment by peterbe** [comment]
Can you add your URL to this when you've got this merged thing into production?
https://github.com/mozilla/contribute.json/blob/master/knownurls.txt
Please.

**Comment by glogiotatidis** [comment]
@tofumatt can we get this to production? I can file the bug according to the readme instructions if you don't mind.

**Comment by tofumatt** [comment]
Yeah, that’d be great. File away! ;-)

**Pull request merged**